sdk/tests/deqp/framework/common/tcuTestCase.js

```diff
@@ -453,7 +453,32 @@ goog.scope(function() {
 
                 if (inited) {
                     // Run the test, save the result.
+
+                    const debug = tcuTestCase._debug = tcuTestCase._debug || (() => {
```

Comment:
Is the double assignment to `tcuTestCase._debug` intentional?

Reply:
It's not double-assignment. The less terse way to write it is:
```
tcuTestCase._debug = tcuTestCase._debug || (() => { // executed if !tcuTestCase._debug };
const debug = tcuTestCase._debug;
```

Reply:
Thanks for explaining.